pkg/groups/handlers.go

```diff
@@ -28,7 +26,7 @@ const (
 )
 
 type UpdateRolesRequest struct {
-	Roles []string `json:"roles" validate:"required"`
+	Roles []string `json:"roles" validate:"required,dive,required"`
```

Comment:
please add a comment saying the double required is needed by `dive`, just until we get familiar, a link to the docs is as good as a sentence happy with anything